gprofiler/main.py

```diff
@@ -382,6 +382,51 @@ def run_continuous(self) -> None:
                     break
 
 
+def send_collapsed_file_only(args, client, enrichment_options):
+    spawn_time = time.time()
+    local_start_time = datetime.datetime.utcnow()
+    monotonic_start_time = time.monotonic()
+    gpid = ""
+    if args.collect_metadata:
+        static_metadata = get_static_metadata(spawn_time=spawn_time, run_args=args.__dict__)
+    metadata = (
+        get_current_metadata(cast(Metadata, static_metadata)) if args.collect_metadata else {"hostname": get_hostname()}
+    )
+    metrics = NoopSystemMetricsMonitor().get_metrics()
+    # TODO:container names man, remember them
+    merged_result, total_samples = concatenate_from_external_file(
+        args.upload_collapsed_file,
+        None,
+        enrichment_options,
+        metadata,
+        metrics,
+    )
+    local_end_time = local_start_time + datetime.timedelta(seconds=(time.monotonic() - monotonic_start_time))
```

Comment:
I think these 2 should also be accepted in the file - so file creator can specify the timeframe. Let's define it as 2 fields in the first line of the collapsed (2 UTC timestamps) and try to load them from there. If not given - use a NULL timestamp (i.e UTC timestamp of the epoch)

Reply:
Done, info in the README.md